.github/workflows/node.js.yml

```
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 13.x, 14.x, 15.x]
+        node-version: [18.x, 19.x, 20.x, 21.x]
```

Actually, let's go with this. 

```suggestion
        node-version: [18.x, 20.4]
```

20.4 makes the tests work again, until the mock-fs bug is fixed. I think it is a good compromise between keeping the tests alive and making sure repoman works on Node.js 20.